.github/workflows/github-pages.yml

```diff
@@ -35,13 +34,13 @@ jobs:
       - name: Gatsby Cache Folder
         uses: actions/cache@v3
         with:
-          key: cache-folder
+          key: cache-folder-${{ matrix.os }}
```

Comment:
Their is no OS matrix in this job

Reply:
Yes. But the caches have this suffix.

Reply:
There is no matrix, so the key evaluates to `cache-folder-`

Reply:
So what should be added here? `cache-folder-Ubuntu`?

Reply:
I believe it would be the `runs-on` value of `ubuntu-latest`